src/OpenTelemetry.Contrib.Instrumentation.EntityFrameworkCore/Implementation/EntityFrameworkDiagnosticListener.cs

```diff
@@ -71,6 +71,17 @@ public override void OnCustom(string name, Activity activity, object payload)
             {
                 case EntityFrameworkCoreCommandCreated:
                     {
+                        if (this.options.SuppressDownstreamInstrumentation)
+                        {
+                            SuppressInstrumentationScope.Enter();
+
+                            // If we are suppressing downstream instrumentation then inject
```

Comment:
> then inject context here

This is not relevant here..

Reply:
Should I change a comment, or remove ‘SuppressInstrumentationScope.Enter’ call?